plugins/flow-visibility/policy-recommendation/antrea_crd.py

```diff
@@ -0,0 +1,577 @@
+# This library is used to define Antrea Network Policy related CRDs in Python.
+# Code structure is following the Kubernetes Python Client library (https://github.com/kubernetes-client/python).
+# Could be improved by using openAPI generator in the future.
```

Comment:
Is this file auto-generated or manually created? If there is a change in Antrea Network Policy related CRD, this file also needs to be changed?

Reply:
Thanks for your review Yiou.
This file is manually created now, yes we need to change this file if there is a related change on Antrea side affecting the policy recommendation. This file could be changed to auto-generated by using openAPI generator in the future like the K8s python lib.